### ISSUE TITLE
Trigger `change` event when value is selected

### ISSUE THREAD

**Issue by mhamann** [AmpersandJS/ampersand-select-view#57]
Presently, it seems that the only way to get a notification when the value of the select view changes is to provide an `update` method on the parent view. This doesn't work well for a view that has multiple select fields or that wants to follow an orchestrator pattern where the child view knows nothing about and does not attempt to call methods on its parent.

We should provide a `change` event trigger whenever `setValue` is called.

**Comment by wraithgar** [comment]
Seems reasonable, and wouldn't take too much to add.

It could be a little cleaner and easier to see what's happening if the trigger happened in the `onChange` function right after `setValue` is called, but this is admittedly bike shedding.

Is this something you're comfortable making a PR for, with a test?

**Comment by mhamann** [comment]
Yep, I can do that. Will try to set aside some time next week to get it written...
